docs/qa/CometBFT-QA-38.md

```diff
@@ -359,8 +359,198 @@ number of transactions processed per minute as compared to the baseline.
+| **8k**       | ![block rate](img38/voteExtensions/08k_1_block_rate.png)      | ![txs rate](img38/voteExtensions/08k_1_total_txs_rate.png)      |
+| **16k**      | ![block rate](img38/voteExtensions/16k_1_block_rate.png)      | ![txs rate](img38/voteExtensions/16k_1_total_txs_rate.png)      |
+| **32k**      | ![block rate](img38/voteExtensions/32k_1_block_rate.png)      | ![txs rate](img38/voteExtensions/32k_1_total_txs_rate.png)      |
+
+
+### Number of rounds
+
+The effect of vote extensions are also felt on the number of rounds needed to reach consensus.
+The following graphs show the number of the highest round required to reach consensus during the whole experiment.
+
+In the baseline and low vote extension lengths, most blocks were agreed upon during round 0.
+As the load increases, more and more rounds were required.
+In the 32k case se see round 5 being reached frequently.
+
+| Experiment   | Number of Rounds per block                                    |
+| ------------ | ------------------------------------------------------------- |
+| **baseline** | ![number of rounds](img38/voteExtensions/baseline_rounds.png) |
+| **2k**       | ![number of rounds](img38/voteExtensions/02k_rounds.png)      |
+| **4k**       | ![number of rounds](img38/voteExtensions/04k_rounds.png)      |
+| **8k**       | ![number of rounds](img38/voteExtensions/08k_rounds.png)      |
+| **16k**      | ![number of rounds](img38/voteExtensions/16k_rounds.png)      |
+| **32k**      | ![number of rounds](img38/voteExtensions/32k_rounds.png)      |
+
+
+We conjecture that the reason is that the timeouts used are inadequate for the extra traffic in the network.
+
+### CPU
+
+The CPU usage reached the same peaks on all tests, but the following graphs show that with larger Vote Extensions, nodes take longer to reduce the CPU usage.
+This could mean that a backlog of processing is forming during the execution of the tests with larger extensions.
+
+
+| Experiment   | CPU                                                   |
+| ------------ | ----------------------------------------------------- |
+| **baseline** | ![cpu-avg](img38/voteExtensions/baseline_avg_cpu.png) |
+| **2k**       | ![cpu-avg](img38/voteExtensions/02k_avg_cpu.png)      |
+| **4k**       | ![cpu-avg](img38/voteExtensions/04k_avg_cpu.png)      |
+| **8k**       | ![cpu-avg](img38/voteExtensions/08k_avg_cpu.png)      |
+| **16k**      | ![cpu-avg](img38/voteExtensions/16k_avg_cpu.png)      |
+| **32k**      | ![cpu-avg](img38/voteExtensions/32k_avg_cpu.png)      |
+
+### Resident Memory
+
+The same conclusion reached for CPU usage may be drawn for the memory.
+That is, that a backlog of work is formed during the tests and catching up (freeing of memory) happens after the test is done.
+
+> :warning:
+> A more worrying trend is that the bottom of the memory usage seems to increase in between runs.
+> This will be investigated in a future issue.
+
```

Comment:
These images show the memory usage over 5 twenty minute runs.
They don't indicate any up trend in memory consumption.

min ![image](https://github.com/cometbft/cometbft/assets/2686194/d6824506-0002-45b3-8805-b05e11d53a36)

max  ![image](https://github.com/cometbft/cometbft/assets/2686194/286c2c14-7ffc-41ea-b88d-7bcce4db926f)

avg 
![image](https://github.com/cometbft/cometbft/assets/2686194/03fbeee3-40a7-4caf-857c-df3edf986c08)

actual values 
![image](https://github.com/cometbft/cometbft/assets/2686194/0964721d-6797-4e3e-a0ad-e180e420da45)

```suggestion
A more worrying trend is that the bottom of the memory usage seems to increase in between runs.
We have investigated this in longer runs and confirmed that there is no such a trend.


```